enhancements/windows-containers/health-management.md

```diff
@@ -224,8 +239,22 @@ For Node configuration WICD will have two separate responsibilities:
 
 #### Bootstrap command
 
-When run with the `bootstrap` command, WICD will do the steps required to ensure that Node is created
-for the instance. This functionality is currently implemented within WMCB with the `init-kubelet` command.
+When run with the `bootstrap` command, WICD will do the steps required to ensure that Node is created for the instance. 
+The `bootstrap` command will start all services that have a `bootstrap` value of true, and exclusively these services.
+
+When containerd is the container runtime (rather than pre-installed Docker), the `bootstrap` command will have the added
+responsibility of starting the containerd service. This is becuase the kubelet service has a dependency on the container
+runtime; the container runtime must have a reachable endpoint (i.e. `\\.\pipe\containerd-containerd` is open and usable)
+when kubelet is initialized (with the `container-runtime` and `container-runtime-endpoint` parameters supplied). 
+The containerd service will be configured based on the information in the services ConfigMap, a priority 0 service with
+no dependencies that points to the eventual location of CNI plug-ins on the instance. Although CNI config is populated
```

Comment:
Why is this the eventual location? Won't it be the location from the get go? It is just that it will be used later by containerd but that should be irrelevant to WICD.

Reply:
CNI config actually gets populated after hybrid-overlay runs, hence the "eventual" wording

Reply:
Why would the location change after hybrid-overlay runs?

Reply:
It won't. Will remove eventual